stellarphot/core.py

```diff
@@ -1529,7 +1543,7 @@ def refcat2(
     clip_by_frame=False,
     padding=100,
     magnitude_limit=None,
-    magnitude_limit_passband="SR",
+    magnitude_limit_passband=None,
```

Comment:
it appears to be defaulting to `None` instead of `SR` as the documentation below implies (and as it formerly did).

Reply:
Later in the code the default is set to SR if `magnitude_limit` is not None...